src/index.js

```diff
@@ -204,7 +204,7 @@ class SWPrecacheWebpackPlugin {
 
     // use the outputFileSystem api to manually write service workers rather than adding to the compilation assets
     return new Promise((resolve, reject) => {
-      outputFileSystem.mkdirp(path.resolve(filepath, '..'), (mkdirErr) => {
+      outputFileSystem.opendir(path.resolve(filepath, '..'), (mkdirErr) => {
```

Comment:
After upgrading node version to support Webpack 5, I noticed this fix started breaking =/

The correct thing to do here is to use `recursive` option from Node > v10.12, as recommended by [mkdirp](https://www.npmjs.com/package/mkdirp#implementation) and as was made on [Webpack](https://github.com/webpack/webpack/pull/10244). This way, the behavior of creating or only reading the folder (when it already exists) is maintained.

```suggestion
      outputFileSystem.mkdir(path.resolve(filepath, '..'), { recursive: true }, (mkdirErr) => {
```